rule.go

```diff
@@ -73,6 +75,12 @@ type Metadata struct {
 	Value string
 }
 
+// Flowbits description.
+type Flowbits struct {
```

Comment:
The type here should be "Flowbit" as this struct represents a singular flowbit, not multiple